ui/push-health/IndividualTests.jsx

```diff
@@ -0,0 +1,233 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Badge,
+  Button,
+  UncontrolledCollapse,
+  Navbar,
+  Nav,
+  NavItem,
+  ButtonGroup,
+  UncontrolledButtonDropdown,
+  DropdownMenu,
+  DropdownToggle,
+  DropdownItem,
+} from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown, faRedo } from '@fortawesome/free-solid-svg-icons';
+
+import Clipboard from '../shared/Clipboard';
+import JobModel from '../models/job';
+
+import TestFailure from './TestFailure';
+
+class IndividualTests extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      clipboardVisible: null,
+      checkedJobs: [],
+      refresh: false, // for refreshing test failure lines
+    };
+  }
+
+  addToCheckJob = (job) => {
```

Comment:
Instead of an `add..` and `remove..`, let's just have a `toggleChecked` function.  You can check if `checkedJobs.includes(job)` and do the right thing accordingly.